metricbeat/module/vsphere/virtualmachine/_meta/data.json

```diff
@@ -1,93 +1,93 @@
 {
-    "@timestamp": "2024-08-12T04:51:25.851Z",
-    "event": {
-        "dataset": "vsphere.virtualmachine",
-        "duration": 115000,
-        "module": "vsphere"
-    },
-    "metricset": {
-        "name": "virtualmachine",
-        "period": 10000
-    },
-    "service": {
-        "address": "127.0.0.1:39149",
-        "type": "vsphere"
-    },
-    "vsphere": {
-      "virtualmachine": {
-        "name": "xt0nmfpv9",
-        "uptime": 5348978,
-        "status": "green",
-        "host": {
-          "id":       "host-32",
-          "hostname": "phx-w1c1-esxi04.com"
+  "@timestamp": "2024-08-12T04:51:25.851Z",
```

Comment:
Can we update the timestamp to be after the create time of snapshots for clarity of users

Reply:
what about the VMs that does not have snapshots?

Reply:
I am just talking about this sample json.
I didn't get your question here

Reply:
updated the json.